test/DynamoCoreTests/WorkspaceSaving.cs

```diff
@@ -778,83 +684,26 @@ public void CustomNodeSaveAsAddsNewCustomNodeToSearchAndItCanBeRefactoredWhilePr
+            var searchResult2 = sm.Search(newName).ToList();
 
             // results are correct
-            Assert.AreEqual(1, ViewModel.SearchViewModel.SearchResults.Count);
-            var node4 = (CustomNodeSearchElement)ViewModel.SearchViewModel.SearchResults[0].Model;
-            Assert.AreEqual(oldId, node4.ID);
-
-        }
-
-        [Test]
-        [Category("Failure")]
-        public void CustomNodeSaveAsAddsNewCustomNodeToSearchAndItCanBeRefactoredWhilePreservingOriginalFromExistingDyf2()
-        {
-            // open custom node
-            // SaveAs
-            // two nodes are returned in search on custom node name, difer 
-
-            var model = ViewModel.Model;
-            var examplePath = Path.Combine(GetTestDirectory(), @"core\custom_node_saving", "Constant2.dyf");
-            ViewModel.OpenCommand.Execute(examplePath);
-
-            var oldId = model.CurrentWorkspace.FirstNodeFromWorkspace<Function>().Definition.FunctionId;
-
-            CustomNodeWorkspaceModel nodeWorkspace;
-            Assert.IsTrue(model.CustomNodeManager.TryGetFunctionWorkspace(oldId, true, out nodeWorkspace));
-
-            var newPath = GetNewFileNameOnTempPath("dyf");
-            var originalNumElements = ViewModel.Model.SearchModel.NumElements;
-
-            // save as
-            nodeWorkspace.SaveAs(newPath, ViewModel.Model.EngineController.LiveRunnerCore); // introduces new function id
-
-            var newId = nodeWorkspace.CustomNodeDefinition.FunctionId;
-
-            // refactor oldId with new name
-
-            CustomNodeWorkspaceModel oldWs;
-            Assert.IsTrue(model.CustomNodeManager.TryGetFunctionWorkspace(oldId, true, out oldWs));
-
-            // refactor oldId with new name
-            oldWs.SetInfo("Constant2 Alt", "TheCat", "TheCat");
-
-            // num elements is unchanged by refactor
-            Assert.AreEqual(originalNumElements + 1, ViewModel.Model.SearchModel.NumElements);
-
-            // search common base name
-            ViewModel.SearchViewModel.SearchAndUpdateResults("Constant2");
-
-            // results are correct
-            Assert.AreEqual(2, ViewModel.SearchViewModel.SearchResults.Count);
-
-            var res1 = ViewModel.SearchViewModel.SearchResults[0];
-            var res2 = ViewModel.SearchViewModel.SearchResults[1];
-
-            Assert.IsAssignableFrom(typeof(CustomNodeSearchElementViewModel), res1);
-            Assert.IsAssignableFrom(typeof(CustomNodeSearchElementViewModel), res2);
-
-            var node1 = (CustomNodeSearchElement)res1.Model;
-            var node2 = (CustomNodeSearchElement)res2.Model;
-
-            Assert.IsTrue((node1.ID == oldId && node2.ID == newId) ||
-                          (node1.ID == newId && node2.ID == oldId));
+            Assert.AreEqual(1, searchResult2.Count);
+            var node4 = searchResult2[0] as CustomNodeSearchElement;
+            Assert.NotNull(node4);
+            Assert.AreEqual(newId, node4.ID);
```

Comment:
This test attempts to test something that's currently not allowable in Dynamo.  That is, users cannot set a custom node name directly without a Save as.  Hence, I don't really see a reason to keep this test around.